plugins/apache/process-compose.yaml

```diff
@@ -0,0 +1,15 @@
+version: "0.5"
+
+processes:
+  apache:
+    command: "echo \"Apache starting on port $HTTPD_PORT\ http://localhost:$HTTPD_PORT\" && apachectl start -f $HTTPD_CONFDIR/httpd.conf -D FOREGROUND"
+    availability:
+      restart: "always"
+  apache-error:
+    command: "tail -f $HTTPD_ERROR_LOG_FILE"
+    availability:
+      restart: "always"
```

Comment:
lets say the user wants to view these error logs, what command should they run to get the output from this `tail -f` ?

Reply:
They can `tail -f` directly or they can use `devbox service manager` and get a UI that looks like this:

![image](https://user-images.githubusercontent.com/544948/220493744-199bae16-4119-4f63-a7da-565a6cbb42ce.png)

Reply:
ah, that's neat